packages/atomic/src/locales.json

```diff
@@ -5573,5 +5573,25 @@
     "zh": "弹出式菜单以获取更多选项卡",
     "zh-cn": "用于更多选项卡的弹出式菜单",
     "zh-tw": "用於更多標籤的快顯式選單"
+  },
```

Comment:
You can run the scripts under `utils/translation-gpt` to fill out the translation for rest of the supported languages. 
Long term I guess we could trigger this auto-generate in CI based in locales.json changes.

Reply:
Wooaaaaah! That's cool!

Reply:
Well it sounds good. There's no readme or anything. How to do you usually run it and starting from which directory. Getting all sorts of ENOENT errors